android-sdk/src/main/java/org/sagebionetworks/bridge/android/BridgeConfig.java

```diff
@@ -161,7 +161,7 @@ public String getAcceptLanguage() {
     }
 
     public int getSdkVersion() {
-        return applicationContext.getResources().getInteger(R.integer.osb_android_sdk_version);
+        return BuildConfig.VERSION_CODE;
```

Comment:
Hmm didn't you switch this to be the context method because it wasn't getting the correct app's version code?